docs/tutorials/coord_sys_tutorial.md

```diff
@@ -0,0 +1,211 @@
+
+```
+                     z up  y front (yaw=0.5*pi)
+                      ^    ^
+                      |   /
+                      |  /
+                      | /
+                      |/
+left (yaw=pi)  ------ 0 ------> x right (yaw=0)
+```
+
+For a box, the value of its yaw angle equals its direction minus a reference direction. In all three coordinate systems in MMDetection3D, the reference direction is always the positive direction of the x-axis, while the direction of a box is defined to be parallel with the x-axis if its yaw angle is 0. The definition of the yaw angle of a box is illustrated in the figure below.
+
+```
+y front
+  ^      box direction (yaw=0.5*pi)
+ /|\        ^
+  |        /|\
+  |     ____|____
+  |    |    |    |
+  |    |    |    |
+__|____|____|____|______\ x right
+  |    |    |    |      /
+  |    |    |    |
+  |    |____|____|
+  |
+```
+
+## Definition of the box dimensions
+
+The definition of the box dimensions cannot be disentangled with the definition of the yaw angle. In last section, we said that the direction of a box is defined to be parallel with the x-axis if its yaw angle is 0. Then naturally, the dimension of a box which corresponds to the x-axis should be <img src="https://render.githubusercontent.com/render/math?math=dx">. However, this is not always the case in some datasets (we will address that later).
+
+The following figures show the meaning of the correspondence between the x-axis and <img src="https://render.githubusercontent.com/render/math?math=dx">, and between the y-axis and <img src="https://render.githubusercontent.com/render/math?math=dy">.
+
+```
+y front
+  ^      box direction (yaw=0.5*pi)
+ /|\        ^
+  |        /|\
+  |     ____|____
+  |    |    |    |
+  |    |    |    | dx
+__|____|____|____|______\ x right
+  |    |    |    |      /
+  |    |    |    |
+  |    |____|____|
+  |         dy
+```
+
+Note that the box direction is always parallel with the edge <img src="https://render.githubusercontent.com/render/math?math=dx">.
+
+```
+y front
+  ^     _________
+ /|\   |    |    |
+  |    |    |    |
+  |    |    |    | dy
+  |    |____|____|____\  box direction (yaw=0)
+  |    |    |    |    /
+__|____|____|____|_________\ x right
+  |    |    |    |         /
+  |    |____|____|
+  |         dx
+  |
+```
+
+## Illustration of the three coordinate systems
+
+The illustration of the three coordinate systems is shown below:
+
+![coord_sys_all](../../resources/coord_sys_all.png)
+
+The three figures above are the 3D coordinate system while the three figures below are the bird's eye view.
+
+## Relation with raw coordinate systems of supported datasets
+
+### KITTI
+
+The raw annotation of KITTI is under Camera coordinate system, see [get_label_anno](../../tools/data_converter/kitti_data_utils.py). In MMDetection3D, to train models on KITTI, the data is first converted from camera to LiDAR, see [get_ann_info](../../mmdet3d/datasets/kitti_dataset.py).
```

Comment:
We should use absolute path instead of relative path since this docs will be viewed on read-the-doc web page? Also change the relative paths below please.